docs/grafana-ui.rst

```diff
@@ -0,0 +1,37 @@
+Grafana-UI
+==========
+
+
+.. todo::
+
+  The following test has been realized on vagrant only - One should list
+  all requirements for reaching the UI using a operationnal cluster.
+
+  On your host, you will need to install kubectl.
```

Comment:
That shouldn't be part of the `todo` note, IIUC. Instead, write it as a requirement, either in plain text, or in a `.. warning::` directive.

Reply:
modified. I tried to encompass all the different infrastructure. (I still specified that we use vagrant)